### DISCUSSION TITLE
Validate tasktype in workflow definition

### DISCUSSION THREAD

**Discussion by dcore94** [Netflix/conductor#3145] [Ideas]
I've spent 3 hours trying to understand why my task did not get delivered even if correctly polled.
At the end the problem was that unfortunately i wrote the type as "simple" instead of "SIMPLE" in the workflow definition.
It shouldn't be hard to validate this or make it case insensitive, is it?
Thanks.

**Comment by apanicker-nflx** [comment]
Yep, we can probably add a validation for this. Since all the other task types can required to be upper-case, lets keep this case sensitive for now, but we can look into removing this restriction across all task types in the future.
